web/src/routes.js

```diff
@@ -4,7 +4,6 @@ import Dashboard from './containers/Dashboard';
 import ApdApplication from './containers/ApdApplication';
 import ApdViewOnly from './containers/viewOnly/Apd';
 import EditAccount from './containers/admin/EditAccount';
-import MyAccount from './containers/admin/MyAccount';
```

Comment:
We probably want to delete this container component, and any associated unit test, too.

Reply:
I think we will end up having a profile page, but we don't have any designs for it yet, so I wanted to leave it there for when we will need it.

Reply:
But, isn't that what git is for?